### ISSUE TITLE
July 13- fine-tuned the model

### ISSUE THREAD

**Issue by perixRoshan** [perixRoshan/crop_pest_detection#8]
On July 13, I fine-tuned the model by adjusting hyperparameters like learning rate and batch size to get better accuracy.
